99-network-custom.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /etc/openwrt_release
+
+wifi_password="TPTlam011205@!"
+ten_wifi="TPT Lam"
```

Comment:
You really have to change your passwords now... You posted them to the upstream openwrt project :-D
Maybe you should also delete this pull request, doesn't look like this was planned